app/services/efile/send_survey_notifications_service.rb

```diff
@@ -0,0 +1,24 @@
+module Efile
+  class SendSurveyNotificationsService
+    include StateFile::SurveyLinksConcern
+    BATCH_SIZE = 5
+
+    def run
+      accepted_submissions = EfileSubmission.joins(:efile_submission_transitions)
```

Comment:
we should add an index for efile submission transitions `to_state`

Reply:
actually since we are just running this once maybe depending on the number of clients this isn't necessary, I would just check the number of clients demo before using. Also I think the batch_size can go up.